### PR TITLE
Fix PagingAndSortingRepository fails for Pageable.unpaged()

### PR DESCRIPTION
**Pull request by upadhyaydhiren**
Modify `SimpleArangoRepository#buildPageableClause` for `Pageable.unpaged()` support. In Existing implementation, `Pageable.unpaged()` use case is not handled properly.

### CODE REVIEW

File: src/main/java/com/arangodb/springframework/repository/SimpleArangoRepository.java [hunk 1]
```diff
@@ -360,7 +362,11 @@ private <S extends T> String buildFilterClause(final Example<S> example, final M
     private String buildPageableClause(final Pageable pageable, final String varName) {
         if (pageable == null) return "";
         Sort persistentSort = AqlUtils.toPersistentSort(pageable.getSort(), mappingContext, domainClass);
```
Comment (aburmeis) [inline src/main/java/com/arangodb/springframework/repository/SimpleArangoRepository.java:364]: with an unpaged pageable, the sort is still ignored, but that is another problem and a rare use case

**Review by aburmeis — APPROVED** [review]

File: src/main/java/com/arangodb/springframework/repository/SimpleArangoRepository.java (hunk 1 above)
Comment (upadhyaydhiren) [reply src/main/java/com/arangodb/springframework/repository/SimpleArangoRepository.java:364]: In `Pageable#unpaged` default sort method implementation have `Sort.unsorted()`. So I didn't  included sort for unpaged `Pageable` in `SimpleArangoRepository#buildPageableClause`

Comment (rashtao) [reply src/main/java/com/arangodb/springframework/repository/SimpleArangoRepository.java:362]: `org.springframework.data.domain.Unpaged` is unsorted, so this fine IMHO

**Review by rashtao — APPROVED** [review]

**Comment by rashtao** [comment]
Hi @upadhyaydhiren ,
please note that we cannot proceed merging this PR until CLA requirements are satisfied, see https://github.com/arangodb/spring-data/pull/255#issuecomment-1399283650

**Comment by upadhyaydhiren** [comment]
> Hi @upadhyaydhiren , please note that we cannot proceed merging this PR until CLA requirements are satisfied, see [#255 (comment)](https://github.com/arangodb/spring-data/pull/255#issuecomment-1399283650)

Hi @rashtao 
I submitted the CLA for about 2 year ago with same email id. Look like some issue with git config but let me check based on cla error what changes required.

You can also suggest, So I will do the needful.

**Comment by upadhyaydhiren** [comment]
> > Hi @upadhyaydhiren , please note that we cannot proceed merging this PR until CLA requirements are satisfied, see [#255 (comment)](https://github.com/arangodb/spring-data/pull/255#issuecomment-1399283650)
> 
> Hi @rashtao I submitted the CLA for about 2 year ago with same email id. Look like some issue with git config but let me check based on cla error what changes required.
> 
> You can also suggest, So I will do the needful.

Hi @rashtao 
I fixed CLA requirement issue. Due to first commit(Email not set properly) its causing issue. I merged into single commit and updated the PR. Please have look

**Comment by rashtao** [comment]
Thanks!

**Pull request merged**